conjureup/download.py

```diff
@@ -116,7 +116,7 @@ def get_remote_url(path):
                "/{}/bundle/{}/archive".format(namespace, bundle))
         return url
 
-    if path.startswith("cs:"):
+    if path.startswith("cs:") or path.startswith("cs:~"):
```

Comment:
Can you give an example of a path that caused a problem here?

If a path starts with 'cs:~' then it also starts with 'cs:', so I don't see how this changes the behavior.